src/Validator.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pest\Symfony;
+
+use function PHPUnit\Framework\assertContains;
+use function PHPUnit\Framework\assertCount;
+use function PHPUnit\Framework\assertEquals;
+use function PHPUnit\Framework\assertNotCount;
+use PHPUnit\Framework\ExpectationFailedException;
+use Symfony\Component\Validator\ConstraintViolationInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+/**
+ * Asserts that $subject doesn't pass validation (or does, if $expected === false).
+ *
+ * @param bool|int|string $expected - int to assert the number of violations,
+ *                                  string to assert it contains a specific violation (format = "propertyPath: message")
+ * @param mixed           $subject  - The variable to validate
+ *
+ * @throws \Exception
+ * @phpstan-ignore-next-line
+ */
+function assertViolated($expected, $subject, ?string $propertyPath = null): void
+{
+    /** @var ValidatorInterface $validator */
+    $validator  = container()->get(ValidatorInterface::class);
```

Comment:
Maybe we should check if Validator Interface exist before and throw a comprehensible extension in case it doesn't ? 

Something like 
```
You need the validator component to use assertViolated assertion. Try run `composer require symfony/validator`
```

Reply:
Definitely 🙂